hrui/js/src/hrui/main.js

```diff
@@ -200,18 +198,15 @@
     /**
      * Misc changes to the page (hiding elements, inserting new ones, etc)
      */
-    function miscContactPageChanges (target) {
+    function miscContactPageChanges () {
       // Hide current employer and job title
       // Contact summary screen:
       $('div.crm-contact-current_employer, div.crm-contact-job_title', '.crm-summary-contactinfo-block').parent('div.crm-summary-row').hide();
       // Inline edit form
-      $('form#ContactInfo input#employer_id, form#ContactInfo input#job_title', target).closest('div.crm-summary-row').hide();
```

Comment:
wasn't `target` important to limit the scope of this selector?

Reply:
I have mentioned the same in the PR doc
> Parameters has been removed from the following functions as that is not necessary, and code works without that.